k2/csrc/ragged_ops.cu

```diff
@@ -1249,14 +1250,62 @@ std::vector<RaggedShape> GetPrefixes(RaggedShape &src,
 
 Ragged<int32_t> AddSuffixToRagged(Ragged<int32_t> &src,
                                   const Array1<int32_t> &suffix) {
-  K2_LOG(FATAL) << "Not Implemented!";
-  return Ragged<int32_t>();
+  NVTX_RANGE(K2_FUNC);
+  K2_CHECK_EQ(suffix.Dim(), src.TotSize(src.NumAxes() - 2));
+  ContextPtr &c = src.Context();
+  Array1<int32_t> dst_values(c, src.NumElements() + suffix.Dim());
+  int32_t *dst_values_data = dst_values.Data();
+  int32_t num_axes = src.NumAxes();
+  const int32_t *src_row_splits_lastm1_data =
+      src.RowSplits(num_axes - 1).Data(),
+                *src_row_ids_lastm1_data = src.RowIds(num_axes - 1).Data(),
```

Comment:
What is the meaning of `m1` in `lastm1`?

Reply:
m1 means "minus 1"

Reply:
I think `src_row_ids_data` is fine here as there are no other `row_ids` here
and there exist no confusions.